### ISSUE TITLE
Question about Semi-sup. Evaluation.

### ISSUE THREAD

**Issue by dingli-dean** [fmthoker/skeleton-contrast#8]
Hello, thx for sharing this great work.
In the repo, I noticed that you have mentioned the experiments about Semi-sup. Evaluation, and the related code for dataset are shown in feeder diretory. However, there still lacks of feeder for this mode, the specific detail is shown as follows:
`from feeder.feeder_downstream_semi_supervised import Feeder`
Line 48 in dataset.py.
Could you share the feeder_downstream_semi_supervised.py file?
Thx again for sharing this work.

**Comment by fmthoker** [comment]
Here is the feeder_downstream_semi_supervised file. You may have to adapt it to make it work.
[feeder_downstream_semi_supervised.txt](https://github.com/fmthoker/skeleton-contrast/files/8539755/feeder_downstream_semi_supervised.txt)

**Comment by wb05025** [comment]
Hi,
Thanks a lot for sharing the semi-supervised feeder. 

I used the feeder and did modification on your action_classification.py according to the description on your paper (train encoder and classifier together). However, the best accuracy I can reach on the cross-sub 60 dataset using 20% semi-rate was 63.4%, which is a bit lower than your paper result (70.8%).

I'm curious if there is any chance you would consider sharing your original codes for semi-supervised classification or your finetuned hyperparameters for this part? Thank you so much!

**Comment by fmthoker** [comment]
The hyperparameters of semi-supervised settings are in the supplementary of the arxiv version (https://arxiv.org/pdf/2108.03656.pdf).
Also, note the reported numbers are based on sequence-representation, so you have to fine-tune the corresponding trained encoder. I hope this helps.
